plugins/lua/sort/places.lua

```diff
@@ -77,6 +85,31 @@ local function get_zone_search_key(zone)
     return table.concat(result, ' ')
 end
 
+local function get_location_search_key(zone)
+    local site = df.global.world.world_data.active_site[0]
+    local result = {}
+
+    -- get language_name and type (we dont need user-given zone name because it does not appear on this page)
+    local building, success, _ = utils.binsearch(site.buildings, zone.location_id, 'id')
```

Comment:
you can leave out the final `, _` since Lua doesn't care if you absorb all the return values from a function. the extras are just dropped.